scripts/firedrake-install

```diff
@@ -756,9 +813,7 @@ def build_update_script():
     with open("firedrake/scripts/firedrake-install", "r") as f:
         update_script = f.read()
 
-    for switch in ["developer", "sudo", "package_manager", "minimal_petsc",
-                   "mpicc", "disable_ssh", "show_petsc_configure_options", "adjoint",
-                   "virtualenv_name"]:
+    for switch in FiredrakeConfiguration._persistent_options:
         update_script = update_script.replace("args.%s = False" % switch,
                                               "args.%s = %r" % (switch, args.__getattribute__(switch)))
```

Comment:
This is put back because it acts as a fall-back when reading/writing to an external JSON fails